ansible/development.yml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: Patch yum sources
+  hosts: manager:acs
+  roles:
+    - yum_patch
+
+- name: Install yum packages
+  hosts: manager
+  remote_user: root
+  roles:
+    - yum
```

Comment:
Not sure about the changes in this file, along with noto.yml, medicina.yml and srt.yml. Some yum packages will fail to install if SELINUX is not yet disabled, I think that at least the yum role should be executed after the selinux one.

Reply:
I have added in the yum roles the git (version >2) installation that is required in "Check presence of dependencies on local repository" step. If the other packages requires SELINUX disabled i can revert the previous order and add just the git installation as preliminary task.

Reply:
done